go-sdk/pkg/sdk/keptn.go

```diff
@@ -257,7 +261,7 @@ func (k *Keptn) createStartedEventForTriggeredEvent(triggeredEvent cloudevents.E
 	return c
 }
 
-func (k *Keptn) createFinishedEventForTriggeredEvent(triggeredEvent cloudevents.Event, eventData interface{}) cloudevents.Event {
+func (k *Keptn) createFinishedEventForTriggeredOrStartedEvent(receivedEvent cloudevents.Event, eventData interface{}, eventSuffix string) cloudevents.Event {
```

Comment:
the name of this method is misleading as it now actually accepts any kind of event suffix

Reply:
`createFinishedEventForReceivedEvent` would be a solution?